rancher/installing-rancher/installing-server/multi-nodes/index.md

```diff
@@ -6,109 +6,55 @@ layout: rancher-default
-      
-6. Point your load balancer at the server targets
-7. Go to new installation ip: `http://<LB ip>:<port>`
-8. Bring up the websocket-proxy:
-   * download the Public Key to verify host tokens:
-     * `curl -X GET -O http(s)://<rancher>/v1/scripts/api.crt`
-   * Bring up websocket-proxy running on the host. (Do not use localhost)
-     * `docker run -d -p <port>:<port> -v $(pwd)/api.crt:/api.crt rancher/server websocket-proxy -jwt-public-key-file=/api.crt -listen-address=0.0.0.0:<port>` 
-9. Bring up `go-machine-service`
-   * Create a service account and API keys:
-      * Visit `http(s)://<rancher_server/v1/accounts` in a web browser.
-      * Click `+Create` button.
-          - kind: service
-          - name: RemoteMachineService
-          - uuid: RemoteMachineService
-      * Click `Show Request` then `Send Request`
-      * Click `Follow Self Link`
-      * In the body there is a links section with a credentials link, click it.
-      * Click `+Create`
-      * Click `Show Request`
-      * Click `Follow Self Link`
-      * Make note of publicValue and secretValue.
-   * Launch `go-machine-service` with this command:
-      
-      ```bash
-      sudo docker run -d --restart=always \
-        -e CATTLE_URL=http(s)://<Rancher URL>/v1 \
-        -e CATTLE_ACCESS_KEY=<Service accounts publicValue> \
-        -e CATTLE_SECRET_KEY=<Service accounts secretValue> \
-        rancher/server go-machine-service
-	  ```	
+* MySQL Database with at least 1 GB RAM
+* External Load Balancer 
+* Nodes to be used in HA setup that meet the single node [requirements]({{site.baseurl}}/rancher/installing-rancher/installing-server/#requirements) 
+
+### Preparing for the High Availability (HA) Setup
+
+1. Prepare a MySQL database with at least 1 GB RAM following the same directions as [starting a single node using an external database]({{site.baseurl}}/rancher/installing-rancher/installing-server/#using-an-external-database), but do not launch Rancher server according to those instructions. By default, users will only be able to access the database from localhost. You will need to grant access to the new user for the network where your Rancher nodes will reside.
+2. Configure an external load balancer that will balance traffic on ports 80 and 443 across a pool of nodes that will be running Rancher server. Depending on your cloud provider, it may be necessary to start the nodes before being able to configure the external load balancer.
+3. Prepare the nodes that will be used in the HA setup. These nodes should meet the same [requirements]({{site.baseurl}}/rancher/installing-rancher/installing-server/#requirements) as a single node setup of Rancher.
+    
+    Currently, our HA setup supports 3 cluster sizes. 
+    * 1 Node: Not really HA
+    * 3 Nodes: Any **one** host can fail
+    * 5 Nodes: Any **two** hosts can fail
```

Comment:
Add a note, that HA setups can be split between data centers connected with high speed low latency links within a region, but should not be attempted across larger geographic regions. When splitting things up within a region, keep in mind that Zookeeper is used under the hood and requires a quorum to stay active. If you split the HA setup between 2 data centers, you will only be able to survive the side with the fewest nodes going down.